src/client/configuration.proto

```diff
@@ -31,4 +31,7 @@ message Configuration {
   repeated Plugin plugins      = 6;
   optional bool disabled       = 7 [ default = false ];
   optional base.proto.Verbosity verbosity = 8;
+
+  optional uint32 connect_timeout = 9 [ default = 10 ];
```

Comment:
Since the config is not binary encoded, it's better to place new field before `send_timeout`, and even renumber all the fields.